### DISCUSSION TITLE
windows 11 24h2 / DWMBlurGlass (2.2.0) çalışmadı.

### DISCUSSION THREAD

**Discussion by IBRAHIMASKAN** [Maplespe/DWMBlurGlass#406] [Q&A]
![Ekran görüntüsü 2024-10-22 234711](https://github.com/user-attachments/assets/b098d37b-f617-46a6-819e-55986abdc00c)

windows 11 24h2 / DWMBlurGlass (2.2.0) did not work. Will there be a new update? windows 11 24h2 error? I just saw this program. Thank you for your hard work.

**Comment by Skyyblaze** [comment]
There's still no update for 24H2. @Maplespe is there perhaps a way to donate some money to you if it makes working on the project more feasible for you again?
